apps/game_backend/lib/game_backend/campaigns/rewards/afk_reward_rate.ex

```diff
@@ -1,16 +1,16 @@
 defmodule GameBackend.Campaigns.Rewards.AfkRewardRate do
   @moduledoc """
-  The representation of a level reward that gives a currency to the user.
+  The representation of a Kaline Tree level reward that gives a currency to the user.
```

Comment:
This is misleading as it sounds like it's a level reward, let's rewrite this so it's clearer!